New log file opened at Sun Jan 15 12:15:49 CET 2017 by 9300@DESKTOP-5PRVS2V
Taking next connection to: 50742
New client attempting to connect on port: 50743-50742
  Address = /192.168.0.12:50742
This client on port 50743 has uuid 57e6367b-e36b-467f-ba90-93a54d96938b configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-57e6367b-e36b-467f-ba90-93a54d96938b
Taking next connection to: 50742
New client attempting to connect on port: 50756-50742
  Address = /192.168.0.12:50742
Handshake exception on socket: 50756
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 50742
New client attempting to connect on port: 50757-50742
  Address = /192.168.0.12:50742
This client on port 50757 has uuid 438a4854-4d83-499e-8c84-d59f1692de67 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-57e6367b-e36b-467f-ba90-93a54d96938b, play-fork-run-438a4854-4d83-499e-8c84-d59f1692de67
Taking next connection to: 50742
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 50742
